package.json

```diff
@@ -75,7 +75,8 @@
         "onLanguage:yaml",
         "onCommand:aws.launchConfigForm",
         "onCommand:aws.toggleSamCodeLenses",
-        "onCommand:aws.addSamDebugConfig"
+        "onCommand:aws.addSamDebugConfig",
+        "onCommand:aws.s3.uploadFileToS3"
```

Comment:
can we reuse the existing `aws.s3.uploadFile` command instead of declaring a new one?